client/src/components/AuthModal/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import Modal from 'react-modal'
+
+import { Icons } from '../../assets/media/icons/Icons'
+
+import { AuthForm } from './AuthForm'
+import { Logo } from './Logo'
+import AuthModalComponent, { customModalStyle } from './styles'
+
+interface AuthModalProps {
+    authModalIsOpen: boolean;
+    setAuthModalIsOpen: (authModalIsOpen: boolean) => void;
+}
+
+export const AuthModal = (props: AuthModalProps) => {
+    const [isLogin, setIsLogin] = useState<boolean>(true)
```

Comment:
for future, I think that it is better to use a different type of variable here. Instead of isLogin, I'd rather use - authStep. I'd create a enum AuthStep = { Login, Registration }, and when I am switching between Login and Registration I'd just change this state with a needed step